recipes/paml/4.9/baseml.ctl

```diff
@@ -0,0 +1,34 @@
+      seqfile = brown.nuc 
+     treefile = brown.trees
+
+      outfile = mlb       * main result file
+        noisy = 2   * 0,1,2,3: how much rubbish on the screen
+      verbose = 0   * 1: detailed output, 0: concise output

```

Comment:
In this case it is fine, we should just be careful to not add to many data to this repo.